x/profiles/keeper/keeper.go

```diff
@@ -205,3 +210,14 @@ func (k Keeper) ValidateProfile(ctx sdk.Context, profile *types.Profile) error {
 
 	return profile.Validate()
 }
+
+// DeleteUnregisteredRelationshipsAndBlocks deletes the relationships and blocks of subspaces-unregistered users
+func (k Keeper) DeleteUnregisteredRelationshipsAndBlocks(ctx sdk.Context) {
```

Comment:
@bragaz This function actually does 3 deletion things, deleting unregistered user relationships, unregistered user blocks and unregistered pairs. I think it can be simplified to `DeleteUnregisteredUserService` and add more comments to explain what the function does. What do you think?

Reply:
In this case I think `DeleteUnregisteredUserReferences` can better summarise what the methods do, please add also a further comment telling it's deleting also the pair